paddle/fluid/pybind/generator_py.cc

```diff
@@ -59,7 +59,7 @@ void BindGenerator(py::module* m_ptr) {
       .def_property("_is_init_py", &framework::Generator::GetIsInitPy,
                     &framework::Generator::SetIsInitPy);
   m.def("default_cpu_generator", &framework::DefaultCPUGenerator);
-  m.def("default_cuda_generator", &framework::getDefaultCUDAGenerator);
+  m.def("default_cuda_generator", &framework::GetDefaultCUDAGenerator);
 }  // end Generator
 }  // end namespace pybind
 }  // namespace paddle
```

Comment:
Maybe not introduced by your PR. It seems the namespace endings are not of the same style.